scipy/stats/_rcont/_rcont.c

```diff
@@ -0,0 +1,215 @@
+/**
+  This file contains two algorithms written in C to generate random two-way
+  tables. The algorithms rcont1 and rcont2 originate from Boyett and Patefield,
+  respectively. For more information, see the docs for each function.
+
+  If you wonder about the spelling, rcont is short for random contingency
+  table. Random contingency table is a bit of a misnomer. The tables generated
+  by these algorithms have no contingency/association between the two
+  variables.
+
+  Author: Hans Dembinski
+*/
+
+#include <math.h>
+
+#include "_rcont.h"
+#include "logfactorial.h"
```

Comment:
```suggestion
#include "logfactorial.h"
#include <math.h>
```

Reply:
The change looks good, thanks! One unrelated failure though :(